ftplugin/dirvish.vim

```diff
@@ -5,6 +5,8 @@ if !hasmapto('<Plug>(dirvish_quit)', 'n')
 endif
 
 nnoremap <buffer><silent> <Plug>(dirvish_up) :<C-U>exe "Dirvish %:h".repeat(":h",v:count1)<CR>
```

Comment:
Aha, I didn't notice that difference.  Now that I see the initial ":h" strips the trailing "/" from the dirvish path, I think the split versions actually need that too.